### DISCUSSION TITLE
Anyone know if it's possible to gather Shipping Template values through the api?

### DISCUSSION THREAD

**Discussion by MyGuyCai** [saleweaver/python-amazon-sp-api#556] [FAQ] [answered]
As the title says, I would like a way to pull all of my shipping templates with all of their data e.g. shipping price, handling time, delivery time etc
I have had a look through the reports endpoint but couldn't see one that would help.
For now I have hard coded the values but if I ever make a change to a shipping template in seller central my repricing values will be wrong. 
Maybe I will just create an entry field and make sure to update them when I make changes to my shipping templates

**Comment by saleweaver** [comment, marked as answer]
I don't think there is a way

**Reply by MyGuyCai** [reply]
Ok, thanks. Will add an input for now and send an email to Amazon support
